src/lib.rs

```diff
@@ -601,6 +601,12 @@ impl Builder {
         self
     }
 
+    /// Sets an explicit path to rustfmt, to use when rustfmt is enabled.
+    pub fn with_rustfmt<P: Into<PathBuf>>(mut self, path: P) -> Self {
```

Comment:
Put this next to the other `rustfmt` related methods/options?

Reply:
Good idea, done!